Bottom: 4d442ff19390926fc2455f1db803a17e2aa31446
Top:    61fcf7911f0a4b78f94f336b5735cadef27be6e8
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-12 17:02:02 +0100

Win32: Select the correct GL libraries to link against


---

diff --git a/configure.ac b/configure.ac
index 74d6e9e..cd696a2 100644
--- a/configure.ac
+++ b/configure.ac
@@ -429,7 +429,7 @@ Either do not use --enable-gl or enable the gtk HID.])
 	esac
 
 	GLU_CFLAGS=
-	GLU_LIBS=-lGLU
+	GLU_LIBS=-lglu32 -lopengl32
 
 	AC_DEFINE([ENABLE_GL], 1,
 		[Define to 1 if GL support is to be compiled in])
